config.json

```diff
@@ -346,6 +346,12 @@
       ]
     },
     {
+      "slug": "spiral-matrix",
+      "difficulty": 5,
```

Comment:
> Maybe difficulty 5 could be a bit too high...

My reasoning was that is seems more difficult than `anagram` and `series`, but  is easier than `bank-account`.

Also, I don't think that exercises should be ordered just based on difficulty, but also to alternate in types, making the experience more enjoyable, so I put it after `food-chain`.

If you have another place/difficulty in mind, please tell me.

> ...if you consider a possible simple recursive solution with shrinking squares? (although your sample solution is of course much more interesting from a theoretical point of view).

I'll curious about how people will approach this exercise...